code/datums/soullink.dm

```diff
@@ -122,6 +122,7 @@
 	if(soulsharer)
 		soulsharer.dust(FALSE)
 
+/datum/soullink/oneway/devilfriend
```

Comment:
Should I leave this subtype here? Right now, the system requires no additional code, but it helps flag what kind of soul link this is

Reply:
I did the same thing with pitchforks, even though two years later, there's still no other types of pitchfork.